examples/ocp_on_aws/ocp_static_data.yml

```diff
@@ -1,8 +1,7 @@
 ---
 generators:
   - OCPGenerator:
-      start_date: 2019-05-01
-      end_date: 2019-07-31
+      start_date: last_month
```

Comment:
You've got the `start_date` for the AWS data as `today`, but the OCP data is `last_month`. I think these should all be `last_month` for the purposes of this document.